CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 
 - Added Makefile to simplify building and developing the project locally. (#67)
 
+- Removed all references to `MainProvider.uploadUrl`. (#72)
+
```

Comment:
v1.5.0 release has been merged (https://github.com/iver-wharf/wharf-web/pull/86). Suggest moving this to a new WIP release of v1.6.0